mne/viz/ica.py

```diff
@@ -277,6 +286,10 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         defaults to [7., 6.].
     show : bool
         Show figure if True.
+    reject : 'auto' | dict | None
+        Allows to use a different rejection parameter to drop epochs. If None,
+        it does not apply a rejection. If 'auto', it applies the rejection
+        fitted with the ICA object if it exists.
```

Comment:
small nitpicks, I'd go with something like this (I tried to signal proposed changes in bold):
Allows to **specify** rejection parameters **used** to drop epochs **(or segments if continuous signal is passed as inst)**. If None, **no rejection is applied**. **The default is** 'auto', **which** applies the rejection **parameters used when fitting** the ICA object.

Reply:
I prefer this one indeed. But just like in the ICA object, in case of inst = Epochs, we don't drop anything, so it is a bit misleading. We should add it in docs I believe

Reply:
So we don't drop epochs on `'auto'` (by default) or never drop epochs? I guess the first option. Yes, that should be mentioned. But that's not very intuitive - drop for raw and not for epochs. Can one use `reject` when fitting ICA on epochs?

Reply:
No, when fitting on epochs, you can't use ```reject```

Reply:
Ok, that makes sense now. :) But the documentation should be clear about that.